packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -809,6 +855,10 @@ class CupertinoSliverNavigationBar extends StatefulWidget {
   /// Defaults to `false`.
   final bool stretch;
 
+  /// The default [drawer], which is a [CupertinoSearchTextField] with some
+  /// padding.
+  static PreferredSizeWidget searchField = const _SearchableCupertinoNavigationBar();
```

Comment:
Wondering if we should use a constructor for this instead of a static member.

Reply:
How would somebody use this and get the value from the search field? It probably should be a constructor.